CHANGELOG.md

```diff
@@ -42,4 +42,4 @@ Breaking changes:
 
 ### v1.0.0 (October 26, 2018)
 
--   This library has been ported from @pxblue/visualizations
+-   This library has been ported from @brightlayer-ui/brightlayer-ui
```

Comment:
The changelog history shouldn't change here. The library for that changelog entry was still ported from @pxblue/visualizations, please revert this change.

Reply:
Done.